plugin/textobj/elixir.vim

```diff
@@ -3,15 +3,19 @@ if exists('g:loaded_textobj_elixir_plugin')
 endif
 let g:loaded_textobj_elixir_plugin = 1
 
-let s:save_cpo = &cpo
-set cpo&vim
+if !exists('g:vim_textobj_elixir_mapping')
+  let g:vim_textobj_elixir_mapping = 'e'
+endif
+
+let s:save_cpo = &cpoptions
+set cpoptions&vim
 
 call textobj#user#plugin('elixir', {
     \ 'any' : {
-    \      'select-a' : 'ae', '*select-a-function*' : 'textobj#elixir#any_select_a',
-    \      'select-i' : 'ie', '*select-i-function*' : 'textobj#elixir#any_select_i',
+    \      'select-a' : 'a' . g:vim_textobj_elixir_mapping, '*select-a-function*' : 'textobj#elixir#any_select_a',
+    \      'select-i' : 'i' . g:vim_textobj_elixir_mapping, '*select-i-function*' : 'textobj#elixir#any_select_i',
     \   },
     \ })
 
-let &cpo = s:save_cpo
+let &cpoptions = s:save_cpo
```

Comment:
You might want to keep the `&cpo` lines unchanged (perhaps that's the author's preferred style?) and minimize the patch.

Reply:
That's a good point. I think the longer names are better for clarity, but keeping the original makes sense.